### DISCUSSION TITLE
[BUG] The cache uses more than 256 GB that was claimed on 1.0.4

### DISCUSSION THREAD

**Discussion by hysci** [Chia-Network/chia-blockchain#3286] [Support]
**Describe the bug**

The cache uses much more than 256.6 GB that was claimed on 1.0.4. Command-line was executed on ubuntu. 

**To Reproduce**

1. git clone chia-blockchain repo, switch to 1.1.2 branch
2. Execute command 
`. ./activate && chia plots create -e -k 32 -b 3390 -r 1 -u 128 -n 1 -t /cache/01 -2 /cache/02 -d /chia`
3. Monitor /cache/02 and /cache/01 with ` while true; do du . --max-depth=1 >> test.log; sleep 600; done`

**Expected behavior**
The cache used at `/cache/01` should be 256.6GB, not >300GB

**Screenshots**

![image](https://user-images.githubusercontent.com/6468488/116772455-cbb3e680-aa81-11eb-96c1-b77dd412cc66.png)

**Desktop (please complete the following information):**

- OS: Ubuntu
- OS Version/Flavor: 20.04
- CPU: AMD EYPC 7302

**Additional context**
Reference:
https://chiadecentral.com/chia-plotting-improvements-in-version-1-04/

**Comment by max918** [comment]
according to Plot Sizes (k-sizes) in Wiki page
create K32 plot should use 332GB during creating temporary space

**Reply by hysci** [reply]
According to release note 1.0.4, the cache required should be reduced to 256 GB. Is the release node wrong? https://github.com/Chia-Network/chia-blockchain/blob/1.0.4/CHANGELOG.md

**Comment by sargonas** [comment]
your command example has `-e` in it, which is the flat to disabled Bitfield Plotting. Doing so generates about 12% or more data, which would account for your size increase. Try it again without the -e (assuming your CPU supports it) and I suspect the results will be more in line with what you expect.

**Reply by hysci** [reply]
I've tested by removing `-e` option, the overall size reduced about 10%, but it didn't reduced to the amount as 1.04 release note said. https://github.com/Chia-Network/chia-blockchain/blob/1.0.4/CHANGELOG.md
Is there something wrong with that release note?

**Reply by sargonas** [reply]
In that screen shot above, is that the output from an `ls` command? I ask because ls outputs file size in bytes... and if my math is right, that output equals 0.3GB, not 300 (which it would equate if it was output in KB, which is why I ask).

If you *are* doing `ls` on the directory, can you try this instead? `ls -lh` which will output the results in human readable format, in either Megabytes of Gigabytes, depending on which is most relevant to the file size.

**Reply by hysci** [reply]
No, it was from `du . --max-depth=1`, so the output was definitely in KiB.

**Reply by hysci** [reply]
I ran 3 jobs simutaneously, and check with `du . --max-depth=1 -h`. Now the cache used is (on ubuntu)
![image](https://user-images.githubusercontent.com/6468488/117294334-19c35280-aea5-11eb-8401-226bd43ea37d.png)

**Reply by hysci** [reply]
I would suspect `Approximate working space used (without final file)` in the log is not the actual maximum cache required.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
